### PR TITLE
Add missing number answer partial

### PR DESCRIPTION
**Pull request by peterdavidhamilton**
## Changes in this PR

Number responses now render and do not cause a 500 error

### Before

500 error

### After
Screenshot 2021-06-02 at 16 54 59

## Next steps

Spec coverage will be handled with ticket #674

### CODE REVIEW

File: app/views/tasks/_long_text_answer.html.erb [hunk 1]
```diff
@@ -7,4 +7,4 @@
     <dd class="govuk-summary-list__value"><%= "-" %></dd>
     <dd class="govuk-summary-list__actions"><%= link_to I18n.t("generic.button.change_answer"), journey_step_path(@journey, step), class: "govuk-link", id: "long-text" %></dd>
   <% end %>
-  </div>
+</div>
```
Comment (jacksonj04) [inline app/views/tasks/_long_text_answer.html.erb:10]: Could we drop these tweaks in unrelated files, or bump them to a separate commit?

**Review by jacksonj04 — APPROVED** [review]

File: app/views/tasks/_long_text_answer.html.erb (hunk 1 above)
Comment (peterdavidhamilton) [reply app/views/tasks/_long_text_answer.html.erb:10]: Sure, I'll address the tidy up in #379
